package.json

```
@@ -56,6 +56,12 @@
       "subtitle": "Tomito",
       "description": "Show timer window",
       "mode": "no-view"
+    },
+    {
+      "name": "toggleTimerWidget",
+      "title": "Toggle Timer Widget",
+      "description": "show or hide the timer widget",
```

```suggestion
      "description": "Show or hide the timer widget",
```
Sentence case for consistency :)